packages/cli/src/lib/utils/cli.ts

```diff
@@ -0,0 +1,46 @@
+import {cli} from 'cli-ux';
+import {buildEvent} from '../../hooks/analytics/eventUtils';
+import {ProcessAbort} from '../errors/processError';
+
+interface ConfirmOptions {
+  /**
+   * Whether to terminate the process when the user does not confirm the action.
+   */
+  exit?: boolean;
+  /**
+   * Used for analytics purposes to name the action. Should not contain sensitive information.
+   * See https://coveord.atlassian.net/wiki/spaces/RD/pages/2214168207/Amplitude for taxonomy
+   */
+  eventName?: string;
+}
+
+/**
+ *
+ * @param question The question requiring a confirmation
+ * @param options
+ */
+export async function confirm(
```

Comment:
I'm not sure I would have added the `exit` property here, it seems a bit of a pollution/overreach/doing too much if you compare to the original `cli.confirm` function ?

I think it would be clearer to just name the `confirm` function to `confirmWithAnalytics`, and have it do only that precisely: Ask a confirmation message, and log UA.

So that as a reader of the code, when you see 

```typescript
confirmWithAnalytics('Yes or no ?', 'the-question-name');
```

It's pretty straightforward to understand what it does without diving into the function implementation.